Week-5/asyncOp.js

```diff
@@ -0,0 +1,71 @@
+const getNumber1 = () =>
+  new Promise((resolve, _reject) => {
```

Comment:
One function with async await and one with generator

Reply:
made changes as discussed